src/other/tablevalidator/demo/tablevalidator.js

```diff
@@ -575,7 +575,7 @@ $document.on( tableParsingCompleteEvent, "#visualoutput > table:eq( 0 )", functi
 	options = $elm.data( ).wbvalidateoptions;
 
 	// Quick Complex Tables Detection
-	if ( options.accessibilty === "headers" ||
+	if ( options.accessibility === "headers" ||
```

Comment:
Even though this option seems like it is only used internally, there might be someone that integrated this tool in their solution. Therefore, we will need to assign the old option name value to the new one if it exists. For example:
```suggestion
	if ( options.accessibilty ) {
		options.accessibility = options.accessibilty;
	}

	if ( options.accessibility === "headers" ||
```

Reply:
The script is under the `demos` folder, so I'm not sure if that's likely, unless they're pulling the code in and modifying it.